Makefile

```diff
@@ -86,78 +86,8 @@ goodfaith_weight = 10
 
 max_extractors = 4
 
-############################# Arabic Wikipedia ################################
-
-datasets/arwiki.sampled_revisions.20k_2016.json:
-	wget -qO- http://quarry.wmflabs.org/run/65713/output/0/json-lines?download=true > $@
-
-datasets/arwiki.autolabeled_revisions.20k_2016.json: \
-		datasets/arwiki.sampled_revisions.20k_2016.json
-	cat $< | \
-	./utility autolabel --host https://ar.wikipedia.org \
-		--trusted-groups=sysop,oversight,editor,bot,rollbacker,checkuser,abusefilter,bureaucrat \
-		--trusted-edits=1000 \
-		--revert-radius=3 \
-		--revert-window=48 \
-		--verbose > $@
-
-datasets/arwiki.revisions_for_review.5k_2016.json: \
-		datasets/arwiki.autolabeled_revisions.20k_2016.json
-	( \
-	 cat $< | \
-	 grep '"needs_review": true' | \
-	 shuf -n 2500; \
-	 cat $< | \
-	 grep '"needs_review": false' | \
-	 shuf -n 2500 \
-	) | shuf > $@
-
-datasets/arwiki.autolabeled_revisions.w_cache.20k_2016.json: \
-		datasets/arwiki.autolabeled_revisions.20k_2016.json
-	cat $< | \
-	revscoring extract \
-		editquality.feature_lists.arwiki.reverted \
-		--host https://ar.wikipedia.org \
-		--extractor $(max_extractors) \
-		--verbose > $@
-
-tuning_reports/arwiki.reverted.md: \
-		datasets/arwiki.autolabeled_revisions.w_cache.20k_2016.json
-	cat $< | \
-	revscoring tune \
-		config/classifiers.params.yaml \
-		editquality.feature_lists.arwiki.reverted \
-		reverted_for_damage \
-		roc_auc.labels.true \
-		--label-weight "true=$(reverted_weight)" \
-		--pop-rate "true=0.035186595582635184" \
-		--pop-rate "false=0.9648134044173649" \
-		--center --scale \
-		--cv-timeout 60 \
-		--debug > $@
-
-models/arwiki.reverted.gradient_boosting.model: \
-		datasets/arwiki.autolabeled_revisions.w_cache.20k_2016.json
-	cat $< | \
-	revscoring cv_train \
-		revscoring.scoring.models.GradientBoosting \
-		editquality.feature_lists.arwiki.reverted \
-		reverted_for_damage \
-		--version $(reverted_major_minor).1 \
-		-p 'max_depth=5' \
-		-p 'learning_rate=0.01' \
-		-p 'max_features="log2"' \
-		-p 'n_estimators=700' \
-		--label-weight "true=$(reverted_weight)" \
-		--pop-rate "true=0.035186595582635184" \
-		--pop-rate "false=0.9648134044173649" \
-		--center --scale > $@
-
-arwiki_models: \
-	models/arwiki.reverted.gradient_boosting.model
-
-arwiki_tuning_reports: \
-	tuning_reports/arwiki.reverted.md
+automated:
```

Comment:
Shucks, I forgot that people using our make system are trying to build specific targets, so we would have to pass the target to the next makefile.

I think what we need is "include Makefile.automated", so that the rules defined there get pulled in as if they were inlined in the main file.